mobile/src/bundles/common/store/actions.ts

```diff
@@ -33,27 +34,23 @@ const createUserDetails = createAsyncThunk<
 });
 
 const updateOnboardingData = createAsyncThunk<
-    UserDetailsGeneralRequestDto,
+    UserDetailsGeneralResponseDto,
     UserDetailsGeneralRequestDto,
     AsyncThunkConfig
 >(`${sliceName}/updateOnboardingData`, async (stepPayload, { extra }) => {
     const { commonApi, notifications } = extra;
-    const { badges, hardSkills, photo, cv, companyLogo, ...payload } =
-        stepPayload;
-    const talentHardSkills = hardSkills?.map((skill) => skill.value);
+    const { badges, photo, cv, companyLogo, ...payload } = stepPayload;
+
+    // if (Object.keys(payload).length === 0) {
+    //     return stepPayload;
+    // }
```

Comment:
remove commented code

Reply:
@katerynakostikova  need it to double check onboarding flow, i will remove it after add badges flow, ok?